examples/spark-with-S3/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# docker file/docker-compose derived from https://github.com/gettyimages/docker-spark
+# https://github.com/gettyimages/docker-spark/blob/master/LICENSE
+FROM debian:stretch
+
+RUN apt-get update
+#    && apt-get install -y curl unzip \
+#    && apt-get install -y python-pkg-resources python3.6 python3.6-venv python3-venv
+
+# Python 3.6
+RUN apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+  libreadline-dev libsqlite3-dev wget curl unzip llvm libncurses5-dev libncursesw5-dev \
+  xz-utils tk-dev libffi-dev liblzma-dev
+RUN curl https://www.python.org/ftp/python/3.6.9/Python-3.6.9.tgz --output /usr/python.tgz
+RUN tar xf /usr/python.tgz -C /usr && rm /usr/python.tgz
+RUN cd /usr/Python-3.6.9 \ 
+ && ./configure --enable-shared \
+ && make -j8 \
+ && make altinstall \
+ && cd -
+
+# Java
+RUN apt-get install -y openjdk-8-jre \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# Hadoop
+ENV HADOOP_VERSION 3.2.1
+ENV HADOOP_HOME /usr/hadoop-$HADOOP_VERSION
+ENV HADOOP_CONF_DIR=$HADOOP_HOME/etc/hadoop
+ENV PATH $PATH:$HADOOP_HOME/bin
+RUN curl -sL --retry 3 \
+  "http://archive.apache.org/dist/hadoop/common/hadoop-$HADOOP_VERSION/hadoop-$HADOOP_VERSION.tar.gz" \
+  | gunzip \
+  | tar -x -C /usr/ \
+ && rm -rf $HADOOP_HOME/share/doc \
+ && chown -R root:root $HADOOP_HOME
+
+# Spark
+ENV SPARK_VERSION 2.4.4
+ENV SPARK_PACKAGE spark-${SPARK_VERSION}-bin-without-hadoop
+ENV SPARK_HOME /usr/spark-${SPARK_VERSION}
+ENV SPARK_DIST_CLASSPATH="$HADOOP_HOME/etc/hadoop/*:$HADOOP_HOME/share/hadoop/common/lib/*:$HADOOP_HOME/share/hadoop/common/*:$HADOOP_HOME/share/hadoop/hdfs/*:$HADOOP_HOME/share/hadoop/hdfs/lib/*:$HADOOP_HOME/share/hadoop/hdfs/*:$HADOOP_HOME/share/hadoop/yarn/lib/*:$HADOOP_HOME/share/hadoop/yarn/*:$HADOOP_HOME/share/hadoop/mapreduce/lib/*:$HADOOP_HOME/share/hadoop/mapreduce/*:$HADOOP_HOME/share/hadoop/tools/lib/*"
+ENV PATH $PATH:${SPARK_HOME}/bin
+RUN curl -sL --retry 3 \
+  "https://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/${SPARK_PACKAGE}.tgz" \
+  | gunzip \
+  | tar x -C /usr/ \
+ && mv /usr/$SPARK_PACKAGE $SPARK_HOME \
+ && chown -R root:root $SPARK_HOME
+
+WORKDIR /cluster-pack
+
+
+
+
```

Comment:
We can keep a single blank line at the end

Reply:
Done